### ISSUE TITLE
link to example is broken 

### ISSUE THREAD

**Issue by ModProg** [JayceFayne/swayipc-command-builder#1]

**Comment by JayceFayne** [comment]
True, there was an example once in [swayipc-rs](https://github.com/JayceFayne/swayipc-rs/commit/82622fb59f9590088f990d5d74e4bfb951d50956#diff-256f49a6334d895cf3f062f143eb91d8aea763773bef0f3e55892e1f0c6394ab)...completely forgot that this repo has no example now that I removed them from swayipc-rs
